pkg/auth/providers/saml/saml_provider.go

```diff
@@ -61,6 +69,21 @@ func Configure(ctx context.Context, mgmtCtx *config.ScaledContext, userMGR user.
 		userType:    name + "_user",
 		groupType:   name + "_group",
 	}
+
+	if samlp.hasLdapGroupSearch() {
+		samlp.ldapProvider = ldap.Configure(ctx, mgmtCtx, userMGR, tokenMGR, name)
+
+		// ignoring errors during boot if not configured
+		ldapConf, certpool, err := samlp.getLDAPConfig()
+		if err != nil {
+			logrus.Infof("getting %s ldap config failed: %s", samlp.name, err)
```

Comment:
it's possible that someone has SAML setup but not ldap, i dont want to log at alll because it could make someone think they have to set this up when in reality it's optional